apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -414,6 +414,33 @@ const PressReleaseListSectionCodec = t.strict({
   sectionID: t.union([t.string, t.null]),
 });
 
+const MediaResourcesItemCodec = t.strict({
+  title: t.string,
+  thumbnail: t.strict({
+    data: t.strict({
+      attributes: t.strict({
+        url: t.string,
+      }),
+    }),
+  }),
+  resource: t.strict({
+    data: t.strict({
+      attributes: t.strict({
+        url: t.string,
+      }),
+    }),
+  }),
+  icon: StrapiImageRequiredSchema,
```

Comment:
```suggestion
```
Strapi will not give the option to upload the icon.

Hardcode it to be `FileDownloadOutlined`